subgraphs/enzyme-core/mappings/v4/KilnStaking.ts

```diff
@@ -1,9 +1,19 @@
-import { arrayUnique } from '@enzymefinance/subgraph-utils';
-import { useKilnStakingPosition } from '../../entities/KilnStakingPosition';
-import { Deposit } from '../../generated/contracts/KilnStaking4Events';
+import { ensureKilnStakingPositionValidator, useKilnStakingPosition } from '../../entities/KilnStakingPosition';
+import { Deposit, ExitRequest } from '../../generated/contracts/KilnStaking4Events';
 
 export function handleDeposit(event: Deposit): void {
   let kilnStakingPosition = useKilnStakingPosition(event.params.caller.toHex());
-  kilnStakingPosition.publicKeys = arrayUnique(kilnStakingPosition.publicKeys.concat([event.params.publicKey]));
-  kilnStakingPosition.save();
+
+  let validator = ensureKilnStakingPositionValidator(event.params.publicKey, kilnStakingPosition);
+  validator.timestamp = event.block.timestamp.toI32();
```

Comment:
the name timestamp is not very descriptive. For an exit request, we have `unstakeSignalledAt`, maybe we could do something similar here, and name it `depositedAt`?

Reply:
Good point. Similar to other entities, I have renamed it to `createdAt`